src/preload.py

```diff
@@ -42,8 +43,9 @@
 GPT_SIZE = SECTOR_SIZE * 34
 MBR_BOOTSTRAP_CODE_SIZE = 446
 
-SPLASH_IMAGE_FROM = "/img/resin-logo.png"
-SPLASH_IMAGE_TO = "/splash/resin-logo.png"
+SPLASH_IMAGE_FROM = "/img/balena-logo.png"
+SPLASH_IMAGE_TO = "/splash/balena-logo.png"
+OLD_SPLASH_IMAGE_TO = "/splash/resin-logo.png"
```

Comment:
These are the possible names of the splash image file in the preloaded balenaOS image.